imagersite/imagersite/templates/generic/home.html

```diff
@@ -11,5 +11,5 @@ <h1>Welcome to Imager!</h1>
     <h2>The site that allows you to post your images online for others to view.</h2>
     <p>If you are not a member, membership is free, so <a href="{% url "registration_register" %}">Signup</a> now.</br></br>If you are already a member please <a href="{% url "auth_login" %}">Signin</a>.</p>
     {% endif %}
-    <img id="honey" src="{% static "Honey_Hug.jpg" %}" alt="">
+    <img id="honey" src="media/images/Honey_Hug.jpg" alt="">
```

Comment:
Since the `MEDIA` directory is being gitignored, you should put this image in the `static` directory so that you always have access to this image.